src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/RequestHeadersLogEnricherOptions.cs

```diff
@@ -1,28 +1,27 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.Diagnostics.CodeAnalysis;
 using Microsoft.Extensions.Compliance.Classification;
 using Microsoft.Shared.DiagnosticIds;
 
-namespace Microsoft.AspNetCore.Diagnostics;
+namespace Microsoft.AspNetCore.Diagnostics.Logging;
 
 /// <summary>
 /// Options for the Request Headers enricher.
 /// </summary>
 public class RequestHeadersLogEnricherOptions
 {
     /// <summary>
-    /// Gets or sets a dictionary of header names that logs should be enriched with and their data classification.
+    /// Gets a dictionary of header names that logs should be enriched with and their data classification.
     /// </summary>
     /// <remarks>
     /// Default value is an empty dictionary.
     /// </remarks>
     [Required]
```

Comment:
I think this can be removed since you deleted its setter

Reply:
```suggestion
```